### DISCUSSION TITLE
Version 1.15.0

### DISCUSSION THREAD

**Discussion by shazzaam7** [xenia-manager/xenia-manager#156] [General]
# Announcement
This is the last major version of Xenia Manager v1. Expect only bugfixes moving forwards as I'm currently in the process of doing a complete rewrite of Xenia Manager for v2.0.0. 
I'm not 100% sure if it will be possible to carry over everything from v1 of Xenia Manager although stuff like content will be just a drag and drop, but you will probably need to add your games again due to the complete overhaul of how Xenia Manager works (Hopefully for the better).
The main reason why I'm doing a complete rewrite is because the code and project structure just isn't good enough or fun to work with. Also I want to the code to follow some guidelines so it's easier for others to contribute to the project (If they want to). I will post more details about the v2 in the Announcements over on the Github Discussions in the near future so be on the lookup for them.
## What's Changed
* Added "Update Patch" option which allows the user to update the patch. Do note that it will reset settings when it updates to the latest version (PR #144)
* Xenia Manager now tracks the play time and it will show the play time when you hover over the boxart (PR #147)
* In the settings, above "Open Xenia Installer" button you can see what Xenia is installed and what version of that specific Xenia is currently installed (PR #149)
* Replaced Xenia Master's game compatibility with Xenia Canary game compatibility which should be much more accurate in terms of compatibility (PR #152)
* Updated some URLs for the game lists and stuff like that (PR #153)
* Fixed an issue with having Mousehook build as Xenia Custom not loading bindings.ini file. This fix should also fix other issues where the "WorkingDirectory" was not properly set when launching Xenia (PR #155)

**Full Changelog**: https://github.com/xenia-manager/xenia-manager/compare/1.14.0...1.15.0

<hr /><em>This discussion was created from the release <a href='https://github.com/xenia-manager/xenia-manager/releases/tag/1.15.0'>Version 1.15.0</a>.</em>

**Comment by ZEROCAUBR** [comment]
![Captura de tela 2024-10-12 025419](https://github.com/user-attachments/assets/60e2bd93-ca7f-4547-a40d-8cdc22de29a2)
Erro ao iniciar jogos pelo app com xenia canary

**Comment by ZEROCAUBR** [comment]
![Gravando 2024-10-12 030153](https://github.com/user-attachments/assets/d25d82f7-3954-4cae-a7b0-07234cec256b)
Não está iniciando nem pelo XENIA MASTER.
